mindsdb/integrations/handlers/strava_handler/strava_handler.py

```diff
@@ -0,0 +1,102 @@
+from mindsdb.integrations.handlers.strava_handler.strava_tables import StravaAllClubsTable
+from mindsdb.integrations.handlers.strava_handler.strava_tables import StravaClubActivitesTable
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+)
+from mindsdb.utilities.log import get_log
+from mindsdb_sql import parse_sql
+
+import requests
+import pandas as pd
+import json
+
+logger = get_log("integrations.stravahandler")
+
+class StravaHandler(APIHandler):
+    """Strava handler implementation"""
+
+    def __init__(self, name=None, **kwargs):
+        """Initialize the Confluence handler.
+        Parameters
+        ----------
+        name : str
+            name of a handler instance
+        """
+        super().__init__(name)
+
+        connection_data = kwargs.get("connection_data", {})
+
+        self.parser = parse_sql
+        self.dialect = 'strava'
```

Comment:
This is non existing dialect, better to remove it

Reply:
Removed it. thanks zoran for mentioning this